chemprop/data/splitting.py

```diff
@@ -8,11 +7,13 @@
 from astartes.molecules import train_test_split_molecules, train_val_test_split_molecules
 from rdkit import Chem
 
-from chemprop.data.datapoints import MoleculeDatapoint
+from chemprop.data.datapoints import MoleculeDatapoint, ReactionDatapoint
 from chemprop.utils.utils import EnumMapping
 
 logger = logging.getLogger(__name__)
-MulticomponentDatapoint = Sequence[MoleculeDatapoint]
+
+Datapoints = Sequence[MoleculeDatapoint] | Sequence[ReactionDatapoint]
+MulticomponentDatapoints = Sequence[Datapoints]
```

Comment:
These could be moved into the top level `types.py` file.

Reply:
Actually they can't be moved to types.py because these types need MoleculeDatapoint but I can't import from chemprop.data in types.py because other modules in chemprop.data import from types.py.